### PR TITLE
Add qrcode for lab reception

### PR DESCRIPTION
**Pull request by morfi11**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "@remult/core": "^1.0.80",
     "@remult/server": "^1.0.80",
     "@remult/server-postgres": "^1.0.80",
+    "angular-qrcode": "^7.2.0",
```
Comment (noam-honig) [inline package.json:46]: don't thing that we need them both

File: src/app/helper-families/helper-families.component.html [hunk 2]
```diff
@@ -177,6 +177,21 @@ <h4 *ngIf="!showLeftFamilies()">
...
+      border-bottom-right-radius: 0;
+      padding-left: 0;
+      padding-right: 0;
+      min-width: 38px;" (click)="prepareMessage(true)">
+      <mat-icon>more_vert</mat-icon>
+    </button>
+    <mat-menu #appMenuSmsReminder="matMenu">
+      <button mat-menu-item (click)="sendWhatsapp()">{{settings.lang.sendLinkOnWhatsapp}}</button>
+      <button mat-menu-item (click)="copyMessage()">{{settings.lang.copyMessageWithLink}}</button>
+      <button mat-menu-item (click)="copyLink()">{{settings.lang.copyLink}}</button>
```
Comment (noam-honig) [inline src/app/helper-families/helper-families.component.html:191]: Why is this needed for a reminder? it's the same link as the send sms

**Comment by morfi11** [comment]
reopen on rebased branch
